Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:21.10
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update -y
+RUN apt install -y git python llvm clang
+
+ENV HOME=/home/foo
```

Comment:
i'm a bit of a docker noob so i'm not sure if this makes sense but it would be really nice if this could default to the user's actual home directory (whatever `~` expands to). is this possible?